mistletoe/html_renderer.py

```diff
@@ -78,7 +78,10 @@ def render_escape_sequence(self, token):
 
     @staticmethod
     def render_raw_text(token):
-        return html.escape(token.content)
+        if getattr(token, 'escape', True):
```

Comment:
Is there a reason for using a `getattr` here, instead of `token.escape`? It seems that in your implementation, all `RawText` tokens will have the `escape` attribute. Or is it a test-related change?